test/utils/create-user-full-flow.ts

```diff
@@ -0,0 +1,93 @@
+import request from 'supertest';
+import { AlkemioClient } from '@alkemio/client-lib';
+import { userData } from './common-params';
+import { TestUser } from './token.helper';
+import { verifyInKratosOrFail } from './kratos/verify-in-kratos-or-fail';
+import { registerInKratosOrFail } from './kratos/register-in-kratos-or-fail';
+
+const PASSWORD = process.env.AUTH_TEST_HARNESS_PASSWORD || '';
+const SERVER_URL = process.env.ALKEMIO_SERVER_URL;
+
+export const registerNewUserInAlkemioOrFail = async (
```

Comment:
why did you copy-pasted this? It's provided in `utils/register-in-alkemio-or-fail.ts`